src/GameLogic/PlayerActions/Skills/TargetedSkillActionBase.cs

```diff
@@ -0,0 +1,22 @@
+﻿// <copyright file="TargetedSkillActionBase.cs" company="MUnique">
+// Licensed under the MIT License. See LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace MUnique.OpenMU.GameLogic.PlayerActions.Skills;
+
+/// <summary>
+/// Action to perform a skill which is explicitly aimed to a target.
+/// </summary>
+public abstract class TargetedSkillActionBase : ITargetedSkillAction
```

Comment:
Why an abstract class without any implementation?

Reply:
This was intended to be a pure virtual class, the default implementation is in the TargetedSkillActionDefault (this is the old one, just added interface)